CMakeLists.txt

```diff
@@ -189,11 +194,13 @@ else()
 	list(APPEND yquake2SDLLinkerFlags ${SDL2_LIBRARY})
 endif()
 
-# We need an OpenGL implementation.
-set(OpenGL_GL_PREFERENCE GLVND)
-find_package(OpenGL REQUIRED)
-list(APPEND yquake2IncludeDirectories ${OPENGL_INCLUDE_DIR})
-list(APPEND yquake2OpenGLLinkerFlags ${OPENGL_LIBRARIES})
+if(GL1_RENDERER OR GL3_RENDERER OR GLES1_RENDERER OR GLES2_RENDERER)
```

Comment:
`GLES2_RENDERER` does not exist, so I assume it should be actually `GLES3_RENDERER`. But still have to mention that the only renderer that requires the OS's OpenGL implementation is GL1; the other three use loaders made with _glad_. Take a look to the `glad*` directories inside `refresh`, they include everything needed to load each OpenGL "version".
So, I believe this could be replaced with `if(GL1_RENDERER)`.